Lil-Peep/src/styles/modules/site-footer.scss

```diff
@@ -1,13 +1,13 @@
 footer {
+  background-color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.